lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'dishes'
+require_relative 'text'
+
+class Order
+  attr_reader :basket, :menu, :dishes, :complete, :text
```

Comment:
I don't think all of these attribute readers are needed. Extra available state from inside our objects can cause problems sometimes - especially if someone else is going to use our code as an API (like a gem) - if we allow them to use more parts of our code, they are likely to rely upon them, and we have to work to maintain that too.